.github/workflows/build-test-package.yml

```diff
@@ -2,141 +2,40 @@ name: Build, test, package
 
 on: [push,pull_request]
 
+env:
+  itk-git-tag: "v5.3rc04"
+  itk-wheel-tag: "v5.3rc04.post2" # Same ITK C++ reference commit, different tag name
```

Comment:
Have these two always referenced the same commit or was the C++ tag updated?

Reply:
The C++ tag is behind -- do we need a newer version?

Reply:
My understanding has been that `v5.3rc04.post2` wheels are tagged at https://github.com/InsightSoftwareConsortium/ITK/commit/835dc01388d22c4b4c9a46b01dbdfe394ec23511 ? This is used in CI for several external modules: [ITKCLEsperanto](https://github.com/InsightSoftwareConsortium/ITKCLEsperanto/blob/master/.github/workflows/build-test-package.yml#L5), [ITKUltrasound](https://github.com/KitwareMedical/ITKUltrasound/blob/master/.github/workflows/build-test-package.yml#L5). Would be good to keep these pointing to the same reference commit to simplify debugging.

Reply:
Ok, version bumped to 835dc01388d22c4b4c9a46b01dbdfe394ec23511.

Reply:
Thanks!